exfat_oal.c

```diff
@@ -128,16 +128,16 @@ static time_t accum_days_in_year[] = {
 
 TIMESTAMP_T *tm_current(TIMESTAMP_T *tp)
 {
-	struct timespec ts;
 	time_t second, day, leap_day, month, year;
 
 #if LINUX_VERSION_CODE < KERNEL_VERSION(4,8,0)
```

Comment:
`ktime_get_real_seconds()` available on Linux kernels lower than 4.8.0 (looks like it was introduced in 3.19.0), is this `#if` really required at all?

Reply:
as an FYI I have a working (built and tested this morning_ PR here: https://github.com/barrybingo/exfat-nofuse/pull/1